api_gateway/src/main/kotlin/org/thechance/api_gateway/endpoints/UserRoute.kt

```diff
@@ -38,7 +40,21 @@ fun Route.userRoutes() {
                 val result = identityService.deleteUser(userId = id, language)
                 respondWithResult(HttpStatusCode.OK, result)
             }
+            post("/{userId}/permission"){
+                val userId = call.parameters["userId"]?.trim().toString()
+                val permission = call.receiveParameters()["permission"]?.trim()?.toInt() ?: 0
```

Comment:
"I think this is the same as the filtering we discussed in the meeting, where we talked about being able to filter by multiple permissions, not just one.

Reply:
this is not a search  just update permission @NadaFeteiha